### PR TITLE
Update to use family-linked webfonts and stop using mixins

### PR DESCRIPTION
**Pull request by Scotchester**
Switches from using our old `u-webfont-` mixins to simple CSS styling, in accordance with the current version of Capital Framework which defaults to family-linked webfonts.

## Changes

- All `u-webfont-` mixins converted to simple CSS styling

## Removals

- `@webfont-` variables that duplicated the ones now in CF.

## Testing

1. Pull branch
1. `grunt`
1. `bundle exec jekyll serve watch`
1. Visit http://localhost:4000/ and browse around; see fonts looking as they should

[Preview this PR without the whitespace changes](?w=0)

## Checklist

* [ ] Changes are limited to a single goal (no scope creep)
* [ ] Code can be automatically merged (no conflicts)
* [ ] Code follows the standards laid out in the [front end playbook](https://github.com/cfpb/front-end)
* [ ] Passes all existing automated tests
* [ ] New functions include new tests
* [ ] New functions are documented (with a description, list of inputs, and expected output)
* [ ] Placeholder code is flagged
* [ ] Visually tested in supported browsers and devices
* [ ] Project documentation has been updated (including the "Unreleased" section of the CHANGELOG)

### CODE REVIEW

**Review by nataliafitzgerald — APPROVED** [review]

File: src/static/css/content.less [hunk 1]
```diff
@@ -533,19 +530,19 @@ figure {
...
   li {
-    .u-webfont-medium;
+    font-weight: 500;
   }
 }
 .regular-ex {
   p,
   li {
-    .u-webfont-regular;
+    font-weight: 400;
```
Comment (jimmynotjim) [inline src/static/css/content.less:545]: Why 400 here and normal in other places?

File: src/static/css/header.less [hunk 2]
```diff
@@ -20,7 +20,7 @@
 			color: @gray-60;
 			.site-org {
 				color: @gray-80;
-				.u-webfont-medium;
+				font-weight: 500;
```
Comment (jimmynotjim) [inline src/static/css/header.less:23]: Totally unrelated to this PR but what's up with the spacing here?

**Comment by jimmynotjim** [comment]
Searched the repo and it looks like you found them all. At this point can we remove all references to the mixins from cf-core?

File: src/static/css/header.less (hunk 2 above)
Comment (Scotchester) [reply src/static/css/header.less:23]: Let's not talk about the quality of the syntax in this repo's Less files. 🙈 🙉 🙊

File: src/static/css/content.less (hunk 1 above)
Comment (Scotchester) [reply src/static/css/content.less:545]: Sloppiness. I meant to use 400 everywhere. Will update.

Comment (Scotchester) [reply src/static/css/content.less:545]: Fixed in 53ee82c

**Comment by Scotchester** [comment]
> Searched the repo and it looks like you found them all. At this point can we remove all references to the mixins from cf-core?

Probably? I'm sure there are projects still using them, but as long as we do a major version update, that shouldn't be a big deal.

**Comment by jimmynotjim** [comment]
> I'm sure there are projects still using them, but as long as we do a major version update, that shouldn't be a big deal. 

Shoot, we should have included this in the v5 release. We'll have to make sure we don't forget when we remove the icon font in v6.

**Review by jimmynotjim — APPROVED** [review]

**Pull request merged**